pom.xml

```diff
@@ -195,6 +195,7 @@
         <dep.frontend-npm.version>11.2.0</dep.frontend-npm.version>
         <dep.gib.version>4.5.4</dep.gib.version>
         <dep.google.http.client.version>1.46.3</dep.google.http.client.version>
+        <dep.guice.version>7.0.1-SNAPSHOT</dep.guice.version>
```

Comment:
Why SNAPSHOT version ?

Reply:
Sorry, it's a mistake :) I didn't clean up my tree

Reply:
Suprisingly snapshot is published